src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/service/TelemetryClientService.kt

```diff
@@ -151,7 +151,9 @@ class TelemetryClientService(
 
     bookingEventAudit?.let {
       flagEventDataMap["visitBooked"] = formatDateTimeToString(it.createTimestamp)
-      flagEventDataMap["actionedBy"] = it.actionedBy
+      flagEventDataMap["actionedBy"] = bookingEventAudit.actionedBy.userName ?: ""
+      flagEventDataMap["bookerReference"] = bookingEventAudit.actionedBy.bookerReference ?: ""
```

Comment:
we could not send these values to appinsights if not null, but nothing defined on the story as such